tests/python/integration/test_scenario.py

```diff
@@ -30,5 +30,10 @@ def test_run_sharepoint_online_site_root_overwrite(user_dss_clients):
 def test_run_sharepoint_online_write_on_empty_root_path(user_dss_clients):
     dss_scenario.run(user_dss_clients, project_key=TEST_PROJECT_KEY, scenario_id="WRITEONEMPTYROOTPATH")
 
+
 def test_run_sharepoint_online_file_overwrite(user_dss_clients):
     dss_scenario.run(user_dss_clients, project_key=TEST_PROJECT_KEY, scenario_id="FILEOVERWRITE")
+
+
+def test_run_sharepoint_online_append_to_list_recipe(user_dss_clients):
```

Comment:
I haven't been able to look at this scenario - can you provide the details or share what it does?

Reply:
It creates a sharepoint list, appends to it and check the resulting number of rows.